gitbook-docs/market/cmd-reference/statement.md

```diff
@@ -1,75 +1,104 @@
 ---
-description: Scripts to manage statements/circuits
+description: Managing circuits and statements
 ---
 
-# statement
+# Statement utilities
 
+This page describes scripts for circuit developers and proof producers,
+`prepare_statement.py` and `statement_tools.py`.
 
+Make sure you've [signed up](user.md) before continuing.
 
-### Prepare Statement
+## Preparing circuit as a statement
 
-Statements/circuits generated by the zkLLVM compiler need to have additional metadata attached to them before they can be published on the proof market.
+Circuits are stored as a *statement* structure on the Proof Market.
+Statement examples can be found in `./example/statements/` directory.
+Attach all additional metadata to circuits generated by the zkLLVM compiler
+before publishing them on the Proof Market.
 
-**Usage**
+### Usage
 
-```
-python3 prepare_statement.py -h 
-```
+To see all parameters that can be passed to the script, run it with the `-h` option:
 
-Output
+```console
+$ python3 prepare_statement.py -h 
 
-```
-usage: prepare_statement.py [-h] -c CIRCUIT -o output file -n NAME -t TYPE
+usage: prepare_statement.py [-h] -c CIRCUIT -o output_file_path -n NAME -t \
+       statement_type (--private | --public)
 
 options:
   -h, --help            show this help message and exit
   -c CIRCUIT, --circuit CIRCUIT
-                        zkllvm compiler output
-  -o output file, --output output file
-                        output file
-  -n NAME, --name NAME  name
-  -t TYPE, --type TYPE  statement type: placeholder-zkllvm or placeholder-vanilla
-
+                        Path to a zkLLVM circuit (*.ll)
+  -o output_file_path, --output output_file_path
+                        Output file
+  -n NAME, --name NAME  Statement name that will be shown on the Proof Market
+  -t statement_type, --type statement_type
+                        Statement type: placeholder-zkllvm or placeholder-vanilla
+  --private             Make a statement private: only accessible by ID
+  --public              Make a statement public: visible on the Proof Market
 ```
 
-Ex:
+Please note that `--private` and `--public` parameters are mutually exclusive:
+* With `--private`, the statement will be accessible only by its ID.
+  For experiments and development purposes, make your statements private.
```

Comment:
```suggestion
  Use private statements for testing circuits in development and making experiments.
```

Reply:
I know that I wrote the original line, but still :)

Reply:
Okay)